matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/model/RoomMemberSummaryEntity.kt

```diff
@@ -39,6 +40,11 @@ internal open class RoomMemberSummaryEntity(@PrimaryKey var primaryKey: String =
             membershipStr = value.name
         }
 
+    var userPresence: UserPresenceEntity? = null
+        set(value) {
+            if (value != field) field = value
+        }
```

Comment:
I'm not sure it make sense to test `!=` for Realm object, @BillCarsonFr WDYT?

Reply:
Yeah, I believe it does not make sense, but I there are other implementations like this in` RoomSummaryEntity`:
`    var latestPreviewableEvent: TimelineEventEntity? = null
        set(value) {
            if (value != field) field = value
        }
`
`    var userDrafts: UserDraftsEntity? = null
        set(value) {
            if (value != field) field = value
        }
`
So if we decide to remove it let's remove it from the other places too, if you agree